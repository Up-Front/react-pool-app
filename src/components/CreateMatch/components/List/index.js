import React from 'react';

const List = props => {
  return (
    <div>
      <h4>choose your victim!</h4>
      <ul>{props.children}</ul>
    </div>
  );
};

export default List;
