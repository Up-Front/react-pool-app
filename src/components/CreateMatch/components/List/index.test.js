import React from 'react';
import { shallow, mount } from 'enzyme';
import List from './';

test('renders without crashing', () => {
  shallow(<List />);
});

test('render List', () => {
  const wrapper = shallow(<List />);
  expect(wrapper).toMatchSnapshot();
});
