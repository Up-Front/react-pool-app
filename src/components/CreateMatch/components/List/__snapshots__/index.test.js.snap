// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render List 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <List />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4>
          choose your victim!
</h4>,
        <ul />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "choose your victim!",
        },
        "ref": null,
        "rendered": "choose your victim!",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4>
            choose your victim!
</h4>,
          <ul />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "choose your victim!",
          },
          "ref": null,
          "rendered": "choose your victim!",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
